integr8ly/integreatly-operator
# [WIP] MGDAPI-4597 enable errcheck check-blank

# Issue link
[MGDAPI-4597](https://issues.redhat.com/browse/MGDAPI-4597)

# What
Enable errcheck check-blank

# Verification steps
- Checkout this PR
- Run `make test/lint`
- Confirm that there are no errors in the output
